sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj

```diff
@@ -3,18 +3,13 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Key Vault Certificates client library</Description>
     <AssemblyTitle>Microsoft Azure.Security.KeyVault.Certificates client library</AssemblyTitle>
-    <Version>4.0.0-preview.7</Version>
+    <Version>4.0.0-preview.8</Version>
     <PackageTags>Microsoft Azure Key Vault Certificates;$(PackageCommonTags)</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-      - Moved Cancel and Delete certificate operation methods from CertificateClient to CertificateOperation.
-      - CertificateClient.DeleteCertificate has been renamed to CertificateClient.StartDeleteCertificate and now returns a DeleteCertificateOperation to track this long-running operation.
-      - CertificateClient.RecoverDeletedCertificate has been renamed to CertificateClient.StartRecoverDeletedCertificate and now returns a RecoverDeletedCertificateOperation to track this long-running operation.
-      - CertificateIssuer.Administrators has been renamed to CertificateIssuer.AdministratorContacts.
-      - CertificateKeyType.Oct has been removed.
-      - The SubjectAlternativeNames class now allows you to add multiple types of subject alternative names using any of the DnsNames, Emails, and UserPrincipalNames collection properties.
-
-      See https://github.com/Azure/azure-sdk-for-net/tree/Azure.Security.KeyVault.Certificates_4.0.0-preview.7/sdk/keyvault/Azure.Security.KeyVault.Certificates/ChangeLog.md for more detailed changes.
+      A small number of parameters were renamed or made required to better align with other languages and to reflect requirements of the Azure Key Vault service.
```

Comment:
A few parameters?
I tend to put the verb first ... Renamed several parameters?
Would it be good to say SDK libraries in other languages for people who don't have full context on the SDK?
Separate out the Made required - was this to align with other languages or to reflect AKV requirements?  If one was one and one is the other, you could have two sentences with fewer clauses.

Reply:
"Made required" - a bit of both. But mainly to align with the service. I'll keep it simple, though.